pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -45,12 +45,6 @@ type ZookeeperClusterSpec struct {
 	// the zookeeper cluster.
 	Labels map[string]string `json:"labels,omitempty"`
 
-	// Size (DEPRECATED) is the expected size of the zookeeper cluster. This
```

Comment:
Removing a field from the ZookeeperCluster object would require regenerating the deep-copy class - `zz_generated.deepcopy.go`. I don't see that being done.

Reply:
done